### DISCUSSION TITLE
Shape comparison for MELA KD Zjj for bkg and signal

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#39] [Comments_HIG_23_001] [answered]
Get normalised to unity plot for signal and backgrounds. The same plot as attached here but instead of stack just comparison plots:

<img width="1221" alt="स्क्रीनशॉट 2023-08-11, अ 5 17 41 पर" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/427a9e73-c7e6-4247-9586-f55b25e2c618">

**Comment by jialin-guo1** [comment]
the plots you put here is not KD_VBF but KD_Zjj and we don't make cut on KD_Zjj. it is just used to make up mass2l2q VS KD_Zjj 2D template

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/ac3a3b40-01bc-4c95-a228-e2b247b76ca3)

merged case

**Reply by ram1123** [reply]
yes, Right.

We need this for both resolved and merged category.

**Reply by jialin-guo1** [reply, marked as answer]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/40c9e0ef-bafa-4afa-8ef8-fcd2e42b9d10)
 
This is resolved case plot.

But before the resolved is like this:
<img width="251" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/fdc52dfc-05a5-4aa2-b06d-4180adfcd6c6">

the change is:
before: 
<img width="1157" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1c50f6fc-1d0f-4c74-8881-a25fa21296a2">

Now: If we find a merged candidate in event, this event will be categorized to merged, otherwise resolved

**Comment by ram1123** [comment]
Please prepare a 1D plot for kD_Zjj with the same bins that are used in this 2D distribution. We also need to add them in AN. I missed adding this information.

It would be good if you can just prepare 1 plot adding signal, and all backgrounds mentioned in the caption of the attached image.

----

![स्क्रीनशॉट 2023-08-16, अ 5 12 22 पर](https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/485b76c0-8281-4e1d-8b60-69b15cf091a6)

**Reply by jialin-guo1** [reply]
there are too many plots with each process, so I add resolved and merged plot in eos web

**find in subdirectories like kd_zjj_zjet**
resolved:
https://guoj.web.cern.ch/guoj/HZZ2L2Q/UL_legacy/note_v4/resolved/2018/

merged
https://guoj.web.cern.ch/guoj/HZZ2L2Q/UL_legacy/note_v4/merged/2018/

**Comment by jialin-guo1** [comment]
# With full mZZ range

![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/75b1a0c5-bf3f-4b68-9610-bc60554717a8)
